react/features/base/conference/actions.js

```diff
@@ -1,5 +1,9 @@
 // @flow
 
+declare var APP: Object;
```

Comment:
I don't think we have an explicit rule about this, but throughout the project we add "declare var..." after the imports.